graylog2-web-interface/src/pages/UnauthorizedErrorPage.jsx

```diff
@@ -8,37 +8,56 @@ import { FetchError } from 'logic/rest/FetchProvider';
 import { Icon, ClipboardButton } from 'components/common';
 import ErrorPage from 'components/errors/ErrorPage';
 
+const createErrorMessageString = (errorDetails: ?string, pageDetails: string, errorMessage: string) => {
+  const defaultText = `${pageDetails}\n${errorMessage}`;
+  if (errorDetails) {
+    return `${errorDetails}\n${defaultText}`;
+  }
+  return defaultText;
+};
+
 type Props = {
+  description?: React.Node,
   error: FetchError,
+  errorDetails?: string,
   location: {
     pathname: string,
   },
+  title: string,
 };
 
-const UnauthorizedErrorPage = ({ error, location: { pathname } }: Props) => {
+const UnauthorizedErrorPage = ({ error, errorDetails, title, description, location: { pathname } }: Props) => {
   const errorMessage = error?.message || JSON.stringify(error);
   const pageDetails = `The permissions check for the following request failed,\nwhile trying to access ${pathname}.`;
-  const description = (
+  const defaultDescription = (
     <>
       <p>You do not have the required permissions to view this resource.</p>
       <p>Please contact your administrator and provide the error details.</p>
     </>
   );
+  const errorMessageString = createErrorMessageString(errorDetails, pageDetails, errorMessage);
   return (
-    <ErrorPage title="Missing Permissions" description={description}>
+    <ErrorPage title={title} description={description || defaultDescription}>
```

Comment:
Whenever we do not want the result of a boolean expression, we should now use `??` instead of `||`. (As long as in this case we do not want to support setting the `description` prop to `null` to explicitly disable showing a description)